docs/developer-guide/integrations/users/keycloak.md

```diff
@@ -0,0 +1,172 @@
+# Keycloak Integrations
+
+## General
+
+MapStore supports various functionalities to integrate with Keycloak:
+
+- [**OpenID support**](../openId/#keycloak): Allows to login to MapStore using a keycloak account.
+- [**Single sign on**](#single-sign-on-integration): Enhances the OpenID support by detecting a session in the keycloak realm and automatically login/logout from MapStore
+- [**Direct user integration**](#direct-user-integration): Enhances the OpenID support making MapStore use keycloak as unique Identity Manager System (IdM), replacing the MapStore DB with Keycloak REST API.
+
+## OpenID
+
+**Keycloak OpenID support** allows to use a keycloak instance as Identity Provider (IdP) via OpenID Connect (OIDC), so the user can login to MapStore using an existing account in keycloak.
```

Comment:
**Keycloak OpenID support** allows to use a keycloak instance as Identity Provider (IdP) via OpenID Connect (OIDC), so that the user can login to MapStore using an existing account in keycloak.

Reply:
```suggestion
**Keycloak OpenID support** allows to use a keycloak instance as Identity Provider (IdP) via OpenID Connect (OIDC), so that the user can login to MapStore using an existing account in keycloak.
```